api/services/images/v1/images.proto

```diff
@@ -54,6 +54,9 @@ service Images {
 
 	// Delete deletes the image by name.
 	rpc Delete(DeleteImageRequest) returns (google.protobuf.Empty);
+
+	// Count the number of images.
+	rpc Count(google.protobuf.Empty) returns (CountImagesResponse);
```

Comment:
Instead of directly having `Count`, can we have something like `Usage` or `Metric` and then have count as a field in that message? I think that will make the interface more cleaner, incase we have to add more metric related option in future.

Reply:
`Usage()` sounds good, but I'm not entirely sure what we'd like to put there.

Reply:
Yeah I wouldn't want to pile in anything other than lightweight calculations in here, and I'm not sure what else could go in there either that would be light-weight, and not have to mix other, non-image data in.

It seems like a `Count()` is good enough here and then a separate service that can pull data in across multiple other services (like what containers are referencing an image, image size (based on content-store), and other types of things.

There just doesn't seem to be anything that could go in here that's just for images.
But if we want to call it Usage and drop a `Total` or `Count` field in there to keep it open for possible future expansion that's probably fine... just not sure what that would ever be.